tests/ui/circulation/test_inhouse_cipo.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2020 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""In house circulation policy tests."""
+
+from copy import deepcopy
+from datetime import datetime
+
+import ciso8601
+import pytz
+
+from rero_ils.modules.items.api import Item
+from rero_ils.modules.items.models import ItemStatus
+from rero_ils.modules.libraries.api import Library
+from rero_ils.modules.loans.api import Loan, LoanAction, LoanState
+
+
+def test_less_than_one_day_checkout(
+        circ_policy_less_than_one_day_martigny,
+        patron_martigny_no_email,
```

Comment:
no need to import `patron_martigny_no_email`

Reply:
Done